react/daily-art-prompt/src/store/rootReducer.test.js

```diff
@@ -447,32 +447,32 @@ describe('indexReducer', () => {
     describe('when action.type equals UPDATE_NEXT_INDEX', () => {
         describe('when there is a next prompt', () => {
             it('should return newIndex', () => {
-                const state = [
-                    {
-                        index: 0
-                    }
-                ]
+                const state =
+                {
+                    index: 0
+                }
                 const action = {
                     type: TYPES.UPDATE_NEXT_INDEX,
                     payload: {
-                        index: state[0].index
+                        index: state.index,
```

Comment:
don't use another variable (state) here, just tell it to have what you want it to have

Reply:
fixed